docs/design.md

```diff
@@ -104,3 +104,14 @@ several ways according to policy:
 
 Then like the usual LibOS pattern it will call the container's entry point as though the container
 was a process unto itself (rather than where it now sits like a library in the LibOS PAL process).
+
+## Local Attestation for Filesystem Decryption Key Provisioning
+
+![EnclaveCC_LA.png](images/EnclaveCC_LA.png)
```

Comment:
Yes, it could be derived in G6, however, we think it could be better to postpone this operation until it is needed with regard to security, performance and resource consumption, thanks.

Reply:
Thanks for the explanation!